src/Microsoft.ML.OnnxTransformer/OnnxCatalog.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.ML
     public static class OnnxCatalog
     {
         /// <summary>
-        /// Applies a pre-trained Onnx model.
+        /// Create a <see cref="OnnxScoringEstimator"/>, which applies a pre-trained Onnx model to the input column.
```

Comment:
> to the input column. [](start = 95, length = 21)

there's no input column param. should we add this? 

How about this: Input/output columns are determined based on the input/output columns of the provided ONNX model.

Reply:
Sounds good!


---
In reply to: [277866453](https://github.com/dotnet/machinelearning/pull/3543#discussion_r277866453) [](ancestors = 277866453)